Source/Scene/TileBoundingVolume.js

```diff
@@ -73,5 +73,12 @@ define([
         DeveloperError.throwInstantiationError();
     };
 
+    /**
+     * Update the bounding volume after the tile is transformed.
+     */
+    TileBoundingVolume.prototype.update = function() {
```

Comment:
Since the sphere and box take different parameters, and the functions are called when the type is known, this function doesn't belong here (nor does it need to be on `TileBoundingRegion.prototype.update`).